src/current/cockroachcloud/aws-privatelink.md

```diff
@@ -1,128 +1,292 @@
+1. In the **Service Name** field, enter the **Service Name** copied from the connection modal in [Step 1](#step-1).
+1. Click **Verify**.
+1. In the **VPC** field, enter the ID of the VPC you want to create your endpoint in.
+1. Verify that the subnets are pre-populated.
+1. In the **Security group** section, select the security group you created in [Step 8](#step-8) and uncheck the box for **default** security group.
+1. Click **Create Endpoint**.
 
-1.  Paste the Endpoint ID you created into the **VPC Endpoint ID** field.
-1.  Click **Verify**.
-1.  {{ site.data.products.db }} will accept the endpoint request. You can confirm the request acceptance by checking if the status is listed as Available on the Amazon VPC Console **Endpoints** page in your AWS account.
+    The VPC Endpoint ID displays.
 
-### Enable private DNS
+1. Copy the Endpoint ID to your clipboard and return to {{ site.data.products.db }}'s **Add PrivateLink** modal.
 
-1.  On the Amazon VPC Console **Endpoints** page, select the endpoint you created.
-1.  Click **Actions**.
-1.  Click **Modify Private DNS Names**.
-1.  Check the **Enable Private DNS Name** checkbox.
-1.  Click **Modify Private DNS Name**.
+</section>
 
-Alternatively, use the AWS CLI to modify the Private DNS Name:
+<section class="filter-content" markdown="1" data-scope="aws-cli">
 
-1.  After the endpoint status changes to Available, run the following AWS CLI command:
+1. Substitute the values from the previous steps and run the following AWS CLI command:
 
     {% include_cached copy-clipboard.html %}
     ~~~ shell
-    $ aws ec2 modify-vpc-endpoint --region $REGION \
-    --private-dns-enabled --vpc-endpoint-id $VPC_ENDPOINT_ID
+    aws ec2 create-vpc-endpoint --region {REGION} \
+                                  --vpc-id {VPC_ID} \
+                                  --subnet-ids {SUBNET_ID1} {SUBNET_ID2} \
+                                  --vpc-endpoint-type Interface \
+                                  --security-group-ids {SECURITY_GROUP_ID1} {SECURITY_GROUP_ID2} \
+                                  --service-name {SERVICE_NAME_PROVIDED_BY_COCKROACH}
     ~~~
 
+1. Locate the VPC Endpoint ID in the CLI output.
+
+1. Copy the Endpoint ID to your clipboard and return to {{ site.data.products.db }}'s **Add PrivateLink** modal.
+
+</section>
+
+</section>
+
+## Step 3. Verify the endpoint ID
+
+1. Click **Next**.
+1. Enter the Endpoint ID, then click **Validate**. If validation fails, check the endpoint ID and try again. Otherwise, click **Next**.
+
+
+1. Click **Next**.
+1. Enter the Endpoint ID, then click **Validate**. If validation fails, check the endpoint ID and try again. Otherwise, click **Next**.
+1. Follow the instructions in the modal to enable **private DNS name** for the endpoint in AWS. When this option is enabled, {{ site.data.products.db }} maintains private DNS records in the VPC for each {{ site.data.products.serverless }} cluster with PrivateLink enabled in the region.
```

Comment:
I see the following for the dedicated tab as well:
> Follow the instructions in the modal to enable private DNS name for the endpoint in AWS. When this option is enabled, CockroachDB Cloud maintains private DNS records in the VPC for each CockroachDB Serverless cluster with PrivateLink enabled in the region.

Reply:
It's only the last sentence that needs to be tweaked for Dedicated, correct?